build.gradle.kts

```diff
@@ -216,6 +215,13 @@ val generate = task("generate") {
     outputs.dir("gen")
 }
 
+runIde {
```

Comment:
Can you move this below to `// Use custom JRE ...`. It configures the same task so you could also put in the check for the `intellijJre` property into the configure block. Right now this is in the middle of the code to configure the code generation